app/src/main/java/nextstep/jwp/http/auth/HttpCookie.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.jwp.http.auth;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class HttpCookie {
+    public static final HttpCookie EMPTY = new HttpCookie(null);
+    private UUID SessionId;
```

Comment:
쿠키가 세션을 유지하기 위해 사용될 수도 있지만, 다른 쿠키도 있을 수 있을 것 같은데, 현재 필드를 보면, 세션아이디를 관리하는 쿠키로 특정되어 있는 것 같아요.

만약 Request의 Cookie에 `yummy_cookie=choco; tasty_cookie=strawberry; JSESSIONID=656cef62-e3c4-40bc-a8df-94732920ed46` 이런식으로 쿠키가 들어있는 경우를 처리해주려면 필드에 `Map<String, String>`을 두거나 하는식으로 관리해야할 것 같아요.

```suggestion
    private final Map<String, String> cookies;
```

Reply:
이 부분은 잘 몰라서 이렇게 두었다는 말이 맞겠네요. cookie에서 `JSESSIONID` 라는 필드값이 있다는 건 알겠지만 쿠키마다 다른 값을 가지는 것인지, 어떠한 필드 값이 있는지에 대한 정보를 제대로 모르는 상태에서 값을 파싱해서 넣는다는 것 자체가 큰 의미가 없을거라 생각했어요. 이러한 값들도 파싱하는 과정에서 넣어줘야 할까요?

Reply:
현재 요구사항에서는 `JSESSIONID` 쿠키 말고는 다른 쿠키들이 쓰일 일은 없긴할 것 같아요
그래도 개인적으로는 `HttpCookie`라는 클래스 자체가 모든 쿠키들을 관리하는? 느낌이 드는데, 구현을 보니 `JSESSIONID`로만 쓰이고 있어서 조금 어색하다는 느낌이 들었어요. 
프론트 - 백을 나눠서 개발할때, 합의하에 쿠키에 데이터를 저장하는 경우도 있을 것 같아서 제 생각에는 3단계 힌트에 주어진 것 처럼 Request 헤더에 담겨오는 쿠키값들을 파싱해서 저장해서 사용하는 게 확장성 면에서도 더 좋을 것 같다는 생각이 드네요😄

Reply:
확장성을 고려하면 확실히 저장하는 방식이 좋아보이네요! 변경했습니다.